### PR TITLE
feat: filter inputs - date time picker input to mantine

### PR DESCRIPTION
**Pull request by IrakliJani**
closes: #7687 

### Description:

CleanShot 2023-10-27 at 12 07 28@2x

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

File: packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx [hunk 1]
```diff
@@ -181,41 +181,29 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
...
+                        // FIXME: remove this once we migrate off of Blueprint
+                        // we are doing type conversion here because Blueprint expects DOM element
+                        // Mantine does not provide a DOM element on onOpen/onClose
+                        popoverProps={{
+                            onOpen: () => popoverProps?.onOpened?.(null as any),
+                            onClose: () => popoverProps?.onClose?.(null as any),
+                        }}
+                        value={rule.values ? rule.values[0] : null}
+                        onChange={(value: Date | null) => {
+                            console.log({ mt: value });
```
Comment (IrakliJani) [inline packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx:198]: reminder for me to remove

File: packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx [hunk 2]
```diff
@@ -181,41 +181,29 @@ const DateFilterInputs = <T extends ConditionalRule = DateFilterRule>(
...
-                        timePrecision={'millisecond'}
-                        formatDate={(value: Date) =>
-                            moment(value).format(`YYYY-MM-DD, HH:mm:ss:SSS`)
-                        }
-                        parseDate={(value) =>
-                            moment(value, `YYYY-MM-DD, HH:mm:ss:SSS`).toDate()
-                        }
-                        onChange={(value: string | null) => {
+                        placeholder={placeholder}
+                        withSeconds
```
Comment (magnew) [inline packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx:187]: We let people set seconds?

**Review by magnew — APPROVED** [review]
Looks good!

File: packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx (hunk 2 above)
Comment (IrakliJani) [reply packages/frontend/src/components/common/Filters/FilterInputs/DateFilterInputs.tsx:187]: we even had milliseconds in BP -  I am removing milliseconds and leaving seconds. maybe one day we'll remove seconds too, who knows 🤷

**Pull request merged**